src/lighteval/tasks/tasks_prompt_formatting.py

```diff
@@ -440,6 +440,26 @@ def ethics_virtue(line, task_name: str = None):
     )
 
 
+def gpqa(line, task_name: str = None):
+    gold_index = random.randint(0, 3)
```

Comment:
is this reproducible ?

Reply:
Should be, but we could set the seed for random at the file level to be sure, I'll add it

Reply:
We set the seed somewhere above in the main function if I remember correctly. However, I'm not sure how seeding works for different modules.

Reply:
Just checked, it works for the whole package.